src/actions/helpers.js

```diff
@@ -62,29 +62,37 @@ export function requestFailure(type: ActionType, error: Client4Error) {
  * @param {...Array<any>} args
  * @returns {ActionFunc} ActionFunc
  */
-export function bindClientFunc(clientFunc: () => Promise<mixed>, request: ActionType,
-    success: ActionType | Array<ActionType>, failure: ActionType, ...args: Array<any>): ActionFunc {
+export function bindClientFunc({
```

Comment:
Should we update the `@param` comments here? Do we need to capture any flow types?

Reply:
Oh yes, I will update the params here. I need to push another commit as i saw few unused flags in mobile repo. So will be removing them and updating this PR along with it.

Reply:
@lieut-data done 👍 Have a look at the flow commit changes. Let me know if you have any comments

Reply:
Aah forgot to change the comment of @param here :(